pkg/azuredx/datasource.go

```diff
@@ -36,7 +35,11 @@ func NewDatasource(settings backend.DataSourceInstanceSettings) (instancemgmt.In
 	}
 	adx.settings = datasourceSettings
 
-	tokenProvider := tokenprovider.NewAccessTokenProvider(tokenCache, datasourceSettings.ClientID, datasourceSettings.TenantID, datasourceSettings.AzureCloud, datasourceSettings.Secret, []string{"https://kusto.kusto.windows.net/.default"})
+	adxScope := "https://kusto.kusto.windows.net/.default"
+	if datasourceSettings.AzureCloud == "govazuremonitor" {
```

Comment:
Please use the constant in clouds.go

Reply:
I changed the implementation accordingly.